pkg/function/server/server.go

```diff
@@ -27,107 +28,59 @@ func New() *server {
 }
 
 // RegisterMapper registers the map operation handler to the server.
-// Example:
-//
-//	func handle(ctx context.Context, keys []string, data functionsdk.Datum) functionsdk.Messages {
-//		_ = data.EventTime() // Event time is available
-//		_ = data.Watermark() // Watermark is available
-//		return functionsdk.MessagesBuilder().Append(functionsdk.NewMessage(data.Value()))
-//	}
-//
-//	func main() {
-//		server.New().RegisterMapper(functionsdk.MapFunc(handle)).Start(context.Background())
-//	}
+// See an example at pkg/function/examples/flatmap/main.go
```

Comment:
i think these examples are based on Godoc guidelines so it will be easy for a casual user reading the doc in the pkg.go.dev.

we should add the example path though so they can dig further.

Reply:
Right. I see these examples available on pkg.go.dev at https://pkg.go.dev/github.com/numaproj/numaflow-go/pkg/function@v0.4.5, which are generated by the `pkg/function/doc.go` instead of this `server.go`. Hence, we might not need it in `server.go`.

Reply:
Sorry, I forgot to publish my [comments](https://github.com/numaproj/numaflow-go/pull/68#discussion_r1287701227) after publishing the PR...

Reply:
There's some misleading here, as a private struct, there's no doc exposed for `server`.